README.md

```diff
@@ -102,6 +102,8 @@ the binary verification command above.
 
 ### Unix / Macintosh
 
+Build Node.js (see below if you just want to build documentation with an existing Node.js):
```

Comment:
I'd rather this line not be modified, it doesn't add enough value to justify the noise. If someone comes looking for how to build docs they will see the section below.